### ISSUE TITLE
Support connecting to docker as a unix domain socket

### ISSUE THREAD

**Issue by Xe** [codetainerapp/codetainer#18]
It is a bad idea to encourage people expose Docker over the network.

**Comment by jandre** [comment]
I agree, it is not ideal. I initially tried to as a Unix domain socket, but had issues using the docker `/container/{id}/attach` API - perhaps it was a user error, or something in the docker library I'm using.  I can look into it some more.
